aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -117,14 +113,10 @@ def resolved_query_string_parameters(self) -> Dict[str, List[str]]:
         This is necessary because different resolvers use different formats to encode
         multi query string parameters.
         """
-        if self.query_string_parameters is not None:
-            query_string = {key: value.split(",") for key, value in self.query_string_parameters.items()}
-            return query_string
-
-        return {}
+        return {k: v.split(",") for k, v in self.query_string_parameters.items()}
 
     @property
-    def resolved_headers_field(self) -> Dict[str, Any]:
+    def resolved_headers_field(self) -> MutableMapping[str, str]:
```

Comment:
Since we're into breaking changes and tidying up, I would argue that `resolved_headers` is a better name for this property. The "field" in the name is redundant, and not even being used in the related `headers` and `multi_value_headers` properties or the similar `resolved_query_string_parameters`.

Reply:
This is a suggestion I'd like you to still consider. Hope it does not get lost by being on top of an outdated commit.